dcos/linkerd-dcos-with-fetch.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "linkerd",
+  "instances": 4,
+  "cpus": 0.25,
+  "mem": 256.0,
+  "acceptedResourceRoles": ["*", "slave_public"],
+  "constraints": [["hostname", "UNIQUE"]],
+  "fetch": [
+    {
+      "uri": "https://<URL TO PRIVATE LINKERD YAML>",
+      "extract": false,
+      "executable": false,
+      "cache": false
+    }
+  ],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/io.buoyant/linkerd/1.3.6/config.yml",
+        "hostPath": "./config.yml",
```

Comment:
i think the `volumes` block is redundant with `fetch`?
looking at https://mesosphere.github.io/marathon/docs/application-basics.html#using-resources-in-applications, i think `fetch` should place `config.yml` in the current directory.

`volumes` seems like another interesting approach to getting a config file to linkerd, but perhaps we call  it out in a separate config?